portfolio/src/main/webapp/script.js

```diff
@@ -32,16 +32,34 @@ function funFactGenerator() {
   factGetter.innerText = fact;
 }
 
-function serverFunFact() {
-    
+/**
+ * Displays a fun fact about Desmond on the page.
+ */
+function serverFunFact() { 
     fetch('/data').then(facts => facts.json()).then(jsonfacts => {
 
-    // Pick a random fact.
-    const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+        // Pick a random fact.
+        const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+
+        // Add it to the page.
+        const factGetter = document.getElementById('factgetter');
```

Comment:
You don't need the factGetter var, try chaining with the next line.

Reply:
Will do .